tunnel/tools/libwg-go/jni.c

```diff
@@ -69,3 +70,25 @@ JNIEXPORT jstring JNICALL Java_com_wireguard_android_backend_GoBackend_wgVersion
 	free(version);
 	return ret;
 }
+
+JNIEXPORT jint JNICALL Java_com_wireguard_android_backend_GoBackend_wgSet(JNIEnv *env, jclass c, jstring ifname, jint handle, jstring settings)
```

Comment:
I'd expect this to log on success as well if the Go backend version does, but perhaps its pointless with the logging RootShell already does.

Reply:
Yeah maybe. Let's wait for Jason's comments on that section before deciding